### ISSUE TITLE
Repository missing tags for nuget package versions

### ISSUE THREAD

**Issue by nadinengland** [epiphanysearch/Epiphany.Core#3]
Although there are 4 versions on Nuget it is unclear which commits they are based on. I propose we use git tags to track them. Going forward it is simple to add the tags, but they can be added retroactively by specifying the commit ID (if known by yourself):

```shell
$ git tag -a v1.0.3 748a469f57300faae6a3c29fab7c4e2300d10bfa -m "Published 2016/12/05"
```

I've used the published date here as the message as added clarity, but this is open for discussion.

**Comment by StefanoChiodino** [comment]
Absolutely. That will be part of the CI integration which I've never set up before, but being able to tell which code is in which Nuget package is quite a must.
